internal/clients/aws.go

```diff
@@ -83,6 +83,14 @@ func TerraformSetupBuilder(version, providerSource, providerVersion string) terr
 		//   e.g. what about setting an assume_role section: https://registry.terraform.io/providers/hashicorp/aws/latest/docs#argument-reference
 		tfCfg := map[string]interface{}{}
 		tfCfg["region"] = awsConf.Region
+		if awsConf.Region == "" {
+			// Some resources, like iam group, do not have a notion of region
```

Comment:
Setting `region` with [this value](https://github.com/crossplane/provider-aws/blob/master/pkg/clients/aws.go#L57) didn't work as well. Terraform returned not a valid region error.